packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/widget/SelectWidget.tsx

```diff
@@ -9,15 +9,15 @@ const filter = createFilterOptions<EnumOptionsType>()
 /**
  * This function finds the enum option that has the corresponding value and returns it.
  *
- * If value is nullish, this fn returns undefined. If there is no corresponding option, then a new object is returned
+ * If value is nullish, this fn returns null. If there is no corresponding option, then a new object is returned
  * where the label and value are set to the provided value.
  */
 export function findValueOption(
   value: string | null | undefined,
   options: EnumOptionsType[],
-): EnumOptionsType | undefined {
+): EnumOptionsType | string | null {
   if (value == null) {
-    return undefined
+    return null
   }
```

Comment:
This was causing issues where MUI `Autocomplete` components were switching from controlled <--> uncontrolled, which triggered console warnings

Reply:
How widespread is this kind of issue?

Reply:
This particular bit of code was only exposed in a few places where we were using RJSF + the QuerySortSelector component used for portals' CardContainer, so not very widespread.

The general "switching between controlled/uncontrolled" warning issue could be happening in many more places, though.